holoviews/core/operation.py

```diff
@@ -126,7 +130,8 @@ def _apply(self, element, key=None):
         for hook in self._postprocess_hooks:
             ret = hook(self, ret, **kwargs)
 
-        if isinstance(ret, Dataset) and isinstance(element, Dataset):
+        if (self._propagate_dataset and
+                isinstance(ret, Dataset) and isinstance(element, Dataset)):
```

Comment:
Odd indentation.

Reply:
Updated to follow PEP8 rules, indentation still the same I think, but looks less awkward now.  Another alternative would be:
```python
if (
    self._propagate_dataset and
    isinstance(ret, Dataset) and
    isinstance(element, Dataset)
):
    ...
```